samples/DispatcherSample.Web/EndsWithStringMatchProcessor.cs

```diff
@@ -19,9 +19,18 @@ public EndsWithStringMatchProcessor(ILogger<EndsWithStringMatchProcessor> logger
 
         public override bool Process(object value)
         {
+            if (value == null)
+            {
+                return false;
+            }
+
             var valueString = Convert.ToString(value, CultureInfo.InvariantCulture);
 
-            var endsWith = valueString.EndsWith(ConstraintArgument);
+            var endsWith = valueString.EndsWith(
+                ConstraintArgument,
+                ignoreCase: true,
+                culture: CultureInfo.InvariantCulture);
```

Comment:
BTW you can use this overload: https://msdn.microsoft.com/en-us/library/ms131430(v=vs.110).aspx

It's a little less verbose.